client/extensions/woocommerce/components/list/list-table/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+class ListTable extends Component {
+
+	render() {
+		const { children } = this.props;
+		return (
+			<Card className="list-table">
+				<table className="list-table__table">
```

Comment:
Tables are not considered evil as long as they are used semantically. In this case table is correct